packages/gatsby-link/src/is-local-link.js

```diff
@@ -0,0 +1,5 @@
+export const isLocalLink = path =>
```

Comment:
Can this be replaced by a regex check instead?

```js
const r = /^(?:[a-z]+:)?\/\//i
return r.test(path)
```

Reply:
I've copy-pasted this from existing code. What would be the benefit of using the regex?

Reply:
using regex it's gonna be only one check and can cover a wide variety of combinations, Example `HTTP:` or `HTTPs` etc. It is also a little bit faster than `startsWith`

Reply:
done